rts/motoko-rts/src/bitrel.rs

```diff
@@ -0,0 +1,107 @@
+//! This module implements a simple subtype cache used by the compiler (in generated code)
```

Comment:
Just a detail, not important: For `make test`, there are some compiler warnings that this is unused. Maybe,`#[allow(dead_code)]` or a simple unit test would resolve this.

Reply:
I first suppressed the warning but then decided adding a unit test would be wiser and, indeed, it revealed a bug that would have been very hard to diagnose later! 

See #3529